src/Model/ExitManager.php

```diff
@@ -46,4 +46,47 @@ public function update(array $exit): bool
 
         return $statement->execute();
     }
+
+    public function exitsFiltered($filter)
+    {
+        if ($filter[0] == []) {
+            $filterByJumpTypes = implode(', ', $filter[1]);
+            $query = "SELECT exit.name, exit.image, exit.department, exit.height, exit.id 
+           from `exit`
+           left join `exit_Has_Type_Jump` on `id_exit`=exit.id
+           left join `type_Jump` on `id_type_jump`=type_jump.id
+           WHERE type_jump.id IN (" . $filterByJumpTypes . ");";
+            return $this->pdo->query($query)->fetchAll();
+        } elseif ($filter[1] == []) {
+            $filterByDepartment = "'" . $filter[0][0] . "'";
+            $filterLength = count($filter[1]);
+            if (count($filter[0]) > 1) {
+                for ($i = 1; $i < $filterLength; $i++) {
+                    $filterByDepartment .=  ", '" . $filter[0][$i] . "'";
+                };
+            };
+            echo $filterByDepartment;
```

Comment:
I know it can be useful to debug but we don't share echos or var_dump in our code.